packages/google_maps_flutter/google_maps_flutter/lib/src/marker.dart

```diff
@@ -164,6 +164,8 @@ class Marker {
     this.visible = true,
     this.zIndex = 0.0,
     this.onTap,
+    this.onDrag,
+    this.onDragStart,
```

Comment:
Could add some tests for these? I think dart side unit tests are possible

Reply:
Hi @cyanglaz, I don't think this can be tested on the dart side as the platform google maps is a mock when running dart tests. Can you confirm or make any suggestions?

I can add and java unit test for the onDrag event as there is already one from onDragStart.